ENH: Increase text size

ElDeveloper opened the pull request:
This patch increases the labels' size by dynamically calculating the
dimensions of the canvas depending on the font and the width of the
text.

Fixes #598

As a comparison:

# Before

![screen shot 2018-02-22 at 12 55 41 pm](https://user-images.githubusercontent.com/375307/36563897-644af93a-17d0-11e8-98c5-8694edb92688.png)


# After

![screen shot 2018-02-22 at 12 55 44 pm](https://user-images.githubusercontent.com/375307/36563901-6794362e-17d0-11e8-87f3-c5a7b84634a6.png)

antgonza commented on emperor/support_files/js/draw.js:
```diff
@@ -122,23 +122,31 @@ define(['underscore', 'three'], function(_, THREE) {
-    canvas.width = size;
-    canvas.height = size;
-    var context = canvas.getContext('2d');
+    var fontSize = 30 * factor, canvas, context, measure;
```
20? Currently looks pretty big.

ElDeveloper commented:
Fair enough, I've adjusted the scale (not the font - canvas rendering works in a rather strange way), updated before and after:

# Before

![screen shot 2018-02-22 at 1 45 05 pm](https://user-images.githubusercontent.com/375307/36565898-b59cd7f8-17d6-11e8-8d7b-6611ef1ce509.png)

# After

![screen shot 2018-02-22 at 1 45 07 pm](https://user-images.githubusercontent.com/375307/36565905-b9ad9d0a-17d6-11e8-83fe-ffbb1546320c.png)

wasade commented:
:+1:
